privacy/views/data_protection_menu_view.xml

```diff
@@ -1,17 +1,14 @@
 <?xml version="1.0"?>
 <!-- Copyright 2018 Eficent Business and IT Consulting Services S.L.
      License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
-<odoo>
 <data>
```

Comment:
Is not `odoo` instead?

Reply:
It is a new info for me since that odoo doesn't have any case using `<data` like a root node.
 - `rgrep -e "^<data" . --include=*.xml -B2`
Output:
 - https://gist.github.com/moylop260/677bc2da91803b7ca99b7fe9b3395460

Maybe they still are supporting an old way like `openerp`.

Self remember, pylint just supporting openerp and odoo tags to get records:
 - https://github.com/OCA/pylint-odoo/blob/b459d4c32ad9c9934c4f2fcb6c71c59515de437a/pylint_odoo/misc.py#L441-L442

Reply:
I have used `<data>` in the past, you can be sure it works. Not sure what's the preferred way, but I guess it's a move to debrand code, like they did when moving `__openerp__.py` to `__manifest__.py` (instead of i.e. `__odoo__.py`). I think it's good to keep it debranded, so we avoid rewriting more things if in the future they rebrand the product yet again.